traffic_ops/traffic_ops_golang/cachesstats/cachesstats.go

```diff
@@ -146,11 +147,29 @@ func addStats(cacheData []CacheData, stats tc.Stats) []CacheData {
 		}
 		bandwidth, ok := stat.Stats[tc.StatNameBandwidth]
 		if ok && len(bandwidth) > 0 {
-			cache.KBPS = bandwidth[0].Val.(uint64)
```

Comment:
I believe this was caused because TM was serving some strings that could not be converted into uint64's. I'm not really sure why TM serves strings for metrics (not really the TM expert), but yeah, definitely sounds like an effort for betterment (unless I'm missing some TM logic).

Reply:
TM serves data as strings because it serves many statistics in a totally unstructured way. String is the only type to which it can be guaranteed that all data can be cast or formatted.

Reply:
I recall watching an Uncle Bob video, I believe around architecture, flexibility, etc. and at one point he had effectively removed all types across a boundary in the system and only relied on strings as the format to pass the data across said boundary. I believe, ultimately, his argument against detractors was (I'm completely paraphrasing here) something along the lines of "So long as you have a full test suite, then the data structure/type will be guarded against".

Anyways, 👍 and can be resolved.